### PR TITLE
chore: update admin users

### PR DESCRIPTION
**Pull request by tpetillot**
**Modifications:**
* Update screen with rework permissions UI

**Ref:** MISC-282

### CODE REVIEW

**Review by Isammoc — APPROVED** [review]

**Comment by ccedric** [comment]
- In the screenshot "Invitation email", it links to localhost:3000. We should get a screenshot from cloud.gatling.io
- A system admin scoped on a team can no longer invite users. Should we add this info?

**Comment by tpetillot** [comment]
@ccedric updated

File: content/reference/admin/users/index.md [hunk 1]
```diff
@@ -18,6 +18,10 @@ To invite a user to your organization, click on the **Invite** button and fill i
 - **Email address**: The email address to which the invitation will be sent. The invited user doesn't have to use the same address in their user profile once connected.
 - **Organization role**: either choose a global role, or select none and specify each role by team according to the permissions you want to grant.
 
+{{< alert warning >}}
+Only global system admin can generate an invitation to an organization
```
Comment (ccedric) [inline content/reference/admin/users/index.md:22]: I'd remove "to an organization"

File: content/reference/admin/users/index.md [hunk 2]
```diff
@@ -18,6 +18,10 @@ To invite a user to your organization, click on the **Invite** button and fill i
 - **Email address**: The email address to which the invitation will be sent. The invited user doesn't have to use the same address in their user profile once connected.
 - **Organization role**: either choose a global role, or select none and specify each role by team according to the permissions you want to grant.
 
+{{< alert warning >}}
+Only global system admin can generate an invitation
```
Comment (ccedric) [inline content/reference/admin/users/index.md:22]: One last review:
I think the following sentence is better worded:
"Only global system admins can generate invitations."

**Pull request merged**